### ISSUE TITLE
Builds fail to upload on macOS 10.12B1

### ISSUE THREAD

**Issue by idyll** [convox/rack#857]
The `convox build` command fails on the macOS 10.12 public beta.
It stalls out at `Uploading`. Specifically the output is:

```
Creating tarball... OK
Uploading...
```

I don't see any logs and since the build isn't started there are no build logs.

**Comment by rcaught** [comment]
Are you sure it's stalled out?  If you on a mediocre Internet connection and have a lot of large files present, or many unnecessary files not being listed in your `.dockerignore` (including `.git`) you could be just still uploading.  #543 would be helpful here.

**Comment by MiguelMoll** [comment]
@rcaught brings up some excellent points. That information would helpful to determine what's going on. I'll look into seeing how the CLI can provide feedback while uploading.

**Comment by idyll** [comment]
My internet connection is 100Mbps up. So it's not that. But I do have more information.

If I leave it long enough. It will succeed. BUT I don't see that on the command line.
The command line is stuck at "Uploading"

**Comment by nzoschke** [comment]
https://github.com/convox/rack/pull/858 is close. Until it gets released you can test it locally by building the CLI off the branch

```
$ git checkout -b origin/upload-status
$ go install github.com/convox/rack/cmd/convox
$ convox build
```

**Comment by ddollar** [comment]
This should be fixed by #948
